Xamarin.Essentials/Launcher/Launcher.uwp.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Threading.Tasks;
+using Windows.System;
+using WinLauncher = Windows.System.Launcher;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Launcher
+    {
+        static async Task<bool> PlatformCanOpenAsync(Uri uri)
+        {
+            var supported = await WinLauncher.QueryUriSupportAsync(uri, LaunchQuerySupportType.Uri);
+            return supported == LaunchQuerySupportStatus.Available;
+        }
+
+        static async Task PlatformOpenAsync(Uri uri)
+        {
+            await WinLauncher.LaunchUriAsync(uri);
```

Comment:
no need to make this async, we should just just return the Task in a =>

Reply:
Done